### DISCUSSION TITLE
zh/config/preview

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#84] [Announcements]
# zh/config/preview

文本类型 要作为文本预览的文件扩展名，用 , 分隔，例如 txt,md,go,tsx。 音频类型 要作为音频预览的文件扩展名，以 , 分隔，例如 mp3,wav,m4a。 视频类型 要作为视频预览的文件扩展名，以 , 分隔，例如 mp4,webm,ogg。 图片类型 要作为图像预览的文件扩展名，以 , 分隔，例如 jpg,jpeg,png,gif,we...

https://alist.nn.ci/zh/config/preview.html

**Comment by jinnan11** [comment]
![markdown](https://s1.ax1x.com/2022/12/01/z07k4A.jpg "markdown")

**Comment by cnkjco** [comment]
苹果手机图片heic的不认

**Comment by IFxmp** [comment]
关于怎么使用自建onlyoffice，教程能写的更详细些吗？目前用docker自建onlyoffice/documentserver服务，但是you_only_office_url这个地址不知道怎么配置。谢谢

**Reply by IFxmp** [reply]
找到教程了
https://github.com/alist-org/alist/discussions/3255

**Reply by IFxmp** [reply]
还有一个
https://github.com/alist-org/alist/discussions/3899

**Reply by IFxmp** [reply]
关于挂载阿里云盘$e_url地址取成阿里云盘直链地址而非alist网站文件地址，造成文件找不到的情况。解决方案就是把$e_url改成$durl就能解决问题。

**Comment by sususu98** [comment]
请问如何修改文本文件的默认预览方式，现在的默认预览方式为Markdown，我想要修改为Text Editor该如何操作？或者是否支持修改默认的预览方式？

**Comment by duzengjie** [comment]
我是在内网部署的，在登录之后会调用两个 获取 js 的 api（一个应该是用于 office 预览的），这两个接口我想不调用，可以通过设置解决吗，还是得改代码。
